frontend/src/styles/common.scss

```diff
@@ -123,10 +123,11 @@ html {
 
 .input-help {
     font-size: 12px;
-    word-break: break-all;
+    word-break:  keep-all;
     color: #adb0bc;
     width: 100%;
     display: inline-block;
+    white-space: normal;
 }
 
 .input-error {
```

Comment:
The modifications made to `.input-help` have some improvements:

1. **word-break**: Changed from `break-all` to `keep-all`. This ensures that words do not get broken across lines, which can improve readability.

2. **white-space**: Added `normal`. This property is typically used with `width`, but here it seems redundant since you already set `display: inline-block`.

Overall, these changes should enhance readability without introducing new issues.